tests/php/PHPStanBearTest.py

```diff
@@ -0,0 +1,23 @@
+from bears.php.PHPStanBear import PHPStanBear
+from coalib.testing.LocalBearTestHelper import verify_local_bear
+
+
+good_file = """
+<?php
+    $a = 1;
+?>
+"""
+
+
+bad_file = """
+<?php
+    $a =
+?>
+"""
+
+
+PHPStanBearTest = verify_local_bear(
```

Comment:
We have been trying to increase test quality and we are trying to move away form verify_local_bear. Please check the bear output to make sure it is outputting the right error.